src/config/dynamic-theme-fixes.config

```diff
@@ -620,6 +620,15 @@ INVERT
 
 ================================
 
+bbb*
+
+CSS
+.scrollableList--Z2s6Her {
```

Comment:
Hmm change it into something like 
```CSS
div[class*="scrolableList"]
```
As after the -- are mostly automatic generated and change over time.

Reply:
Thanks for the tip, I'm just a beginner with CSS.